README.md

```diff
@@ -350,9 +341,15 @@ Your API should now be running locally on your port 8002. Access your automatica
 
 ```python
 import requests
+
+headers = {"accept": "application/json"}
+params = {"det_arch": "db_resnet50", "reco_arch": "crnn_vgg16_bn"}
+
 with open('/path/to/your/doc.jpg', 'rb') as f:
-    data = f.read()
-response = requests.post("http://localhost:8002/ocr", files={'file': data}).json()
+    files = [  # application/pdf, image/jpeg, image/png supported
+        ("files", ("doc.jpg", f.read(), "image/jpeg")),
+    ]
+print(requests.post("http://localhost:8080/ocr", headers=headers, params=params, files=files).json())
```

Comment:
In my experience, it's better to have the shortest snippet possible for users to try it out. We went from 3 LoC to 5 excluding formatting, and this PR doesn't change the minimal backend API so I'm confused as to why there is a change in the "files" passed :thinking: (if that was changed in another PR, I think we should avoid putting it in this PR :sweat_smile: )

A few suggestions:
- either we identify the N°1 thing people want to modulate in our snippet (say model or file type) and we pass explicitly without changing much (the path to the file in my opinion)
- or we stick to the original version

Questions/notes:
- there is a typo above where we pass the port as env var (doesn't work actually)
- is the accept header really necessary here? params aren't JSON data (it formats the URL) and I don't think files are as well
- are the params necessary? Let's perhaps leave the default?
- assuming some people will copy paste, I'd suggest storing the result in `response` and optionally print it so that people don't have to run it again to get the response

This is a minor detail in the scope of the PR, but maintaining accessibility & readibility of snippet is important for adoption :)

Reply:
Correct the API has changed in #1522 (with multi file and params support)

Yeah i agree that has nothing to do with the PR (it's something i missed with the API change PR) 

1. Let me check about the port :)
2. and 3.  params & header are optional but about the params i think this shows well that a user can paramterize the request !?
4. Yeah sure :)

Reply:
Alright, the new syntax is a bit complex for file handling :sweat_smile: 

Only one final thing: I did use the right port but when uploading a file using this snippet, I'm getting errors on the backend side saying it cannot access the `.content_type` of the `UploadFile`. I haven't the other PR, so I don't know whether this is some fastapi/starlette version issue or a backend issue but I think there is a problem (if you managed to run it locally, it's quite likely this is a version constraint issue)